frontend/package.json

```diff
@@ -10,7 +10,9 @@
 		"lint": "eslint . --debug",
 		"lint:fix": "eslint . --fix --debug",
 		"cypress:open": "cypress open",
-		"cypress:run": "cypress run"
+		"cypress:run": "cypress run",
+		"test": "jest",
+		"test:coverage": "jest --coverage"
```

Comment:
Also would be great if you can add watch script also which would be helpful for local development